lib/utils.ts

```diff
@@ -51,3 +52,25 @@ export class MissingParamError extends Error {
     this.message = `${param} is required`;
   }
 }
+
+export const verifyWebhookSignature = (
+  webhookSigningSecret: string,
+  timestamp: string,
+  signature: string,
+  payload: Buffer,
+): boolean => {
+  if (isEmpty(webhookSigningSecret) || isEmpty(timestamp) || isEmpty(signature) || isEmpty(payload)) {
+    throw new MissingParamError('webhookSigningSecret, timestamp, signature, payload');
+  }
+
+  const hmac = createHmac('sha256', webhookSigningSecret);
+  hmac.update(`v0:${timestamp}:`);
+  hmac.update(payload);
+
+  const hash = hmac.digest();
+  if (!timingSafeEqual(hash, Buffer.from(signature, 'hex'))) {
+    return false;
+  }
+
+  return true;
```

Comment:
I think we can simplify this if the function returns a boolean.

```suggestion
  return timingSafeEqual(hash, Buffer.from(signature, 'hex'));
```

Reply:
Yeah, that's true. Copied it from our internal function, and there we had some extra logs.